site/js/modules/webui/render.js

```diff
@@ -401,10 +402,7 @@ function submitInputs(params, context, container, inputs, onSubmitFunc, postResu
             continue;
         }
 
-        let value = result.getFieldValue();
-        if ((input.type === Input.INPUT_TYPE_BOOL) || ((value) && (value != ""))) {
-            inputParams[result.getFieldName()] = value;
-        }
+        inputParams[result.getFieldName()] = result.getFieldValue();
```

Comment:
These semantics are important and need to be discussed somewhere.
Maybe in the function comment for makePage() around the discussion of `onSubmitFunc`.